ncm-fstab/src/main/perl/fstab.pm

```diff
@@ -38,7 +38,7 @@ sub update_entries
         my $el2 = $el->getNextElement();
         my $fs = NCM::Filesystem->new ($el2->getPath()->toString(), $config);
         $self->debug (4, "Update fstab entry at $fs->{mountpoint}");
```

Comment:
ah - good point. So I think the wording of this message should be changed to "Checking" not "updating". Did we ever agree on what levels of debug should be used? The fact that /etc/fstab has been modified will get logged when we fix the logging in CAF::FileWriter to say what files it touched.